### PR TITLE
Fix on action list

### PR DESCRIPTION
**Pull request by brunitob**
Now we do correct order with transactions 

image

### CODE REVIEW

File: src/hooks/usePrimitivesFromSmartVault.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+              id
+              type
+              token {
+                id
+                name
+                symbol
+                decimals
+              	}
+              amount
+            	}
```
Comment (facuspagnuolo) [inline src/hooks/usePrimitivesFromSmartVault.js:92]:
```suggestion
            movements {
              id
              type
              token {
                id
                name
                symbol
                decimals
              }
              amount
            }
```

File: src/hooks/usePrimitivesFromSmartVault.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+              target	
+            	sender
+              gasUsed
+              gasPrice
+              costNative
+              costUsd
+              relayer
+              hash
+            }
+
```
Comment (facuspagnuolo) [inline src/hooks/usePrimitivesFromSmartVault.js:70]:
```suggestion
            }
```

File: src/hooks/usePrimitivesFromSmartVault.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  //TODO: put id in the query. Cause for some reason is failing
+  let data = await request(
+    CHAIN_SUBGRAPH_URL[chainId],
+    gql`
+    {
+          primitiveExecutions(orderBy: transaction__executedAt, orderDirection: desc, where: {smartVault: ${'"' + id.toLowerCase() + '"'}}){
+            id
+      			smartVault {
+              totalValueManaged
+            }
```
Comment (facuspagnuolo) [inline src/hooks/usePrimitivesFromSmartVault.js:55]:
```suggestion
            smartVault {
              totalValueManaged
            }
```

File: src/hooks/usePrimitivesFromSmartVault.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+      totalValueManaged: data?.totalValueManaged || 0,
+      lastAction: actions && actions[0] && actions[0][1],
+      actions: actions && actions?.slice(0, limit),
+      isLoading: isLoading
+    }
+  }, [data, isLoading, limit])
+}
+
+const fetchSmartVault = async (chainId, id) => {
+  //TODO: put id in the query. Cause for some reason is failing
```
Comment (facuspagnuolo) [inline src/hooks/usePrimitivesFromSmartVault.js:46]: Is there any issue with that actually? I think it's fine to have the smart vault id there

File: src/hooks/usePrimitivesFromSmartVault.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+              totalValueManaged
+            }
+            type
+            data
+            transaction {
+              id
+              executedAt
+              target	
+            	sender
+              gasUsed
```
Comment (facuspagnuolo) [inline src/hooks/usePrimitivesFromSmartVault.js:63]:
```suggestion
              target	
              sender
              gasUsed
```

File: src/pages/ActionHistory.js [hunk 6]
```diff
@@ -10,17 +10,19 @@ import Action from '../sections/Action'
...
   // TODO: delete limit when actionPage is ready
-  const smartVault = useSmartVaultWithPrimitives(id, 10000)
+  // const smartVault = useSmartVaultWithPrimitives(id, 10000)
+  const smartVault = usePrimitivesFromSmartVault(id, 10000)
 
   return (
     <Page>
-      {!smartVault.isLoading && !smartVault?.id ? (
+      {/* {!smartVault.isLoading && !smartVault?.id ? ( */}
+      {false ? (
```
Comment (facuspagnuolo) [inline src/pages/ActionHistory.js:25]: Was this intended?

Comment (brunitob) [reply src/pages/ActionHistory.js]: nop, good catch

File: src/hooks/usePrimitivesFromSmartVault.js (hunk 4 above)
Comment (brunitob) [reply src/hooks/usePrimitivesFromSmartVault.js:46]: Legacy. Solved ✅

**Pull request merged**